app/components/notification-list/template.hbs

```diff
@@ -0,0 +1,27 @@
+<div class="container">
+    <div class="row">
+        <div class="notification-list col-md-8">
+            <div class="notification-list-heading-container">
+                <h4 class="notification-list-heading">{{t 'components.notificationList.heading'}}</h4>
+                <span class="helper-text">
```

Comment:
If this is a `div` instead of a `span` it will keep it's padding when wrapping:
![screen shot 2018-04-02 at 2 03 42 pm](https://user-images.githubusercontent.com/7131985/38208355-cc76b298-367e-11e8-9bf4-cb097d9926eb.png)